pkg/plugin/aggregation/run_test.go

```diff
@@ -152,7 +153,9 @@ func TestRunAndMonitorPlugin(t *testing.T) {
 			}
 
 			go func() {
-				a.RunAndMonitorPlugin(ctx, testTimeout, tc.plugin, fclient, nil, "testname", testCert, &corev1.Pod{}, "", "/tmp/sonobuoy/results")
+				var waitGroup sync.WaitGroup
```

Comment:
You typically try to keep the waitgroup logic all within the same function rather than passing it around. Is there a reason why you want to pass it into that function rather than just calling the wg.Done() after something gets sent to the done channel?

Reply:
Hm I think I see why you did it. Maybe we can talk about the options on a zoom call. Since RunAndMonitor would normally be synchronous, but we are calling it with `go RunAndMonitor...` we could do something slightly different to avoid passing in the WaitGroup (which requires a dev to follow the code path to figure out what happens to it). Instead we could do:

```
go func() {
  a.RunAndMonitorPlugin(...)
  waitGroup.Done()
}()
```
so the method signature doesnt have to change and in the same functions we see when the waitGroup gets marked as done.

I could be mistaken, but that seems a slightly better option. Its a small improvement though and wont seriously change the approach or tests.